server/chat-commands/info.ts

```diff
@@ -803,7 +803,7 @@ export const commands: Chat.ChatCommands = {
 						Gen: String(ability.gen) || 'CAP',
 					};
 					if (ability.isPermanent) details["&#10003; Not affected by Gastro Acid"] = "";
-					if (ability.isUnbreakable) details["&#10003; Not affected by Mold Breaker"] = "";
+					if (ability.ignorable) details["&#10003; Ignored by Mold Breaker"] = "";
```

Comment:
What's with the flip? Are there more abilities affected by Mold Breaker or not affected?

Reply:
(Out of the abilities that could potentially be affected by Mold Breaker.)

Reply:
oh its flipped because isUnbreakable will be deleted

Reply:
I'm saying why should it be deleted?

Reply:
I think it'd be better if it was flipped back.